package.json

```diff
@@ -7,10 +7,10 @@
     "test": "lerna run test",
     "postinstall": "husky install",
     "boot": "yarn clean:lib && yarn build:packages --stream",
-    "storybook": "start-storybook -p 6006",
+    "storybook": "start-storybook -p 6006 -s .storybook/public",
```

Comment:
<img width="813" alt="Screenshot 2022-01-20 at 19 27 03" src="https://user-images.githubusercontent.com/26967723/150399270-5ae0f337-c001-4a55-bb55-f02122dd8c82.png">

https://storybook.js.org/docs/react/configure/images-and-assets#serving-static-files-via-storybook